cinema/urls.py

```diff
@@ -0,0 +1,8 @@
+from django.urls import path
+from cinema.views import cinema_list, cinema_details
+
+urlpatterns = [
+    path("cinema/movies/", cinema_list, name="cinema_list"),
+    path("cinema/<int:pk>", cinema_details, name="cinema_detail"),
```

Comment:
Ensure that all your endpoints end with a `/`. According to the checklist, the path should be `path("cinema/<int:pk>/", cinema_details, name="cinema_detail")` to ensure consistency and avoid potential issues with URL matching.